picard/ui/statusindicator.py

```diff
@@ -74,7 +137,7 @@ def set_progress(self, progress):
 
     class WindowsTaskbarStatusIndicator(AbstractProgressStatusIndicator):
         def __init__(self, window):
-            super().__init__()
+            super().__init__(window)
```

Comment:
Is this change related to the PR?

Reply:
Part of the janky hack commit. Thinking on how to make it better, but getting a good ETA is my priority.